.github/workflows/release.yml

```diff
@@ -0,0 +1,152 @@
+name: Create release and upload to Apple and Google
+on:
+  push:
+    tags:
+      # Only builds for tags with a meaningless build number suffix: v1.0.0-1
+      - 'v[0-9]+.[0-9]+.[0-9]*-.*'
+
+jobs:
+  build:
+    name: Build ios and android package
+    runs-on: macos-latest
+
+    steps:
+      - name: Set up Go 1.19.1
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.19.1
+
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Install flutter
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.3.2'
+
+      - uses: nttld/setup-ndk@v1
+        id: setup-ndk
+        with:
+          ndk-version: r21b
+          # r21b is 21.1.6352462, if it is not published here https://developer.android.com/ndk/downloads
+          # or here https://github.com/android/ndk/wiki/Unsupported-Downloads then you must download them and look at
+          # source.properties to determine the version
+
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Configure git for private modules
+        env:
+          TOKEN: ${{ secrets.MACHINE_USER_PAT }}
+        run: git config --global url."https://defined-machine:${TOKEN}@github.com".insteadOf "https://github.com"
+
+      - name: Install the appstore connect key material
+        env:
+          AC_API_KEY_SECRET_BASE64: ${{ secrets.AC_API_KEY_SECRET_BASE64 }}
+        run: |
+          AC_API_KEY_SECRET_PATH="$RUNNER_TEMP/key.p8"
+          echo "APP_STORE_CONNECT_API_KEY_KEY_FILEPATH=$AC_API_KEY_SECRET_PATH" >> $GITHUB_ENV
+          echo -n "$AC_API_KEY_SECRET_BASE64" | base64 --decode --output "$AC_API_KEY_SECRET_PATH"
```

Comment:
Does this work? GNU base64 doesn't support `--output` AFAIK

Reply:
Works on a mac :D